src/services/formatting/smartIndenter.ts

```diff
@@ -729,6 +729,11 @@ export namespace SmartIndenter {
                     return false;
                 }
                 break;
+            case SyntaxKind.TryStatement:

```

Comment:
After reading this, I was really hoping that putting `TryStatement` above with `ForStatement` but that did not work either.

Reply:
Yes, an explicit check is needed to ensure finally isn't treated as a child of the try statement